quartz/styles/base.scss

```diff
@@ -76,6 +77,16 @@ a {
       padding: 0;
     }
   }
+
+  &.external .external-icon {
+    text-align: bottom;
```

Comment:
i don't think `bottom` is a valid value for `text-align` https://developer.mozilla.org/en-US/docs/Web/CSS/text-align

maybe https://developer.mozilla.org/en-US/docs/Web/CSS/vertical-align ?

Reply:
You're right, probably hit the wrong autocomplete.